### DISCUSSION TITLE
Acme plugin : Max Domain Limit , Efficiency.

### DISCUSSION THREAD

**Discussion by atul161** [Kong/kong#8102] [Help]
Hii ,

I have just gone through the acme plugin and found that we can use admin api for creating global acme plugin. Few questions i just wanna ask is : 
 
  - We can add multiple domain while installing acme plugin using admin api. How can we update domain name or add new domain name in that plugin configuration.
  - How feasible is it to maintain a list of array of domains?
  - Is there any option to add domain name at runtime or by admin api and remove exact domain name from that array using admin api?
  - Suppose we have 10000 of dns then how feasable is it to maintain the list of array? Do we have any other approaches?
  - Is there any upper limit of domain name array?
  
  Thankyou!!

**Comment by fffonion** [comment]
@atul161 Right now Kong doesn't support substract/add on an array, you can use tools like https://github.com/kong/deck or the underlying go client to do some fancy logic. Also if your domain names share the same parent domain, consider using wildcard
`*`. The upper limit of domain name array is capped by your datastore/database.
